spec/v1/CSN-Interop-Effective.schema.yaml

```diff
@@ -554,6 +555,8 @@ definitions:
           - "cds.LargeString"
           - "cds.Integer"
           - "cds.Integer64"
```

Comment:
TODO: Clarify - should this have been `cds.Int64` ? Then we can also go with `cds.Int16`.

Then we'd be consistent with https://cap.cloud.sap/docs/guides/domain-modeling#standard-built-in-types